packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -89,6 +89,40 @@ export const makeBidSpecShape = (currencyBrand, collateralBrand) => {
 
 /** @typedef {import('@agoric/vat-data').Baggage} Baggage */
 
+/**
+ * @typedef {object} PriceLevel
+ * A description of a single quote
+ *
+ * @property {Amount<'nat'>} amountIn
+ * The amount supplied to a trade
+ * @property {Amount<'nat'>} amountOut
+ * The quoted result of trading `amountIn`
+ */
+
+/**
+ * @typedef {object} BookDataNotification
+ *
+ * @property {PriceQuote | null} startPrice identifies the priceAuthority and price
```

Comment:
a little hard to read. consider column aligning like https://github.com/Agoric/agoric-sdk/blob/c93516ed1f45452d2286dccc75b08befeef6e4de/packages/inter-protocol/src/vaultFactory/vaultManager.js#L78-L81